lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,31 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # defines class Car
+  class Car
+    attr_reader :color, :model, :year, :current_speed
+    # initialize for Car object
+    def initialize(car_praremeters)
+      @year = car_praremeters[:year] || 2015
+      @color = car_praremeters[:color] || 'purple'
+      @model = car_praremeters[:model] || 'skyline'
+      @current_speed = 0
+    end
+
+    # create new Car object with default parameters
+    def self.default_car
```

Comment:
According to style guide, class methods have to be placed before instance methods